Integrations/python/deephaven/types.py

```diff
@@ -0,0 +1,29 @@
+# 
+# Copyright (c) 2016 - 2021 Deephaven Data Labs and Patent Pending
+# 
+
+"""
+Functions to return types from jpy for convenience
+"""
+
+import jpy
+
+# Raise a SystemError if jpy is not aware of the JVM yet
+if not jpy.has_jvm():
+    raise SystemError("No java functionality can be used until the JVM has been initialized through the jpy module")
```

Comment:
Confirming that if you start a clean console and run the following that everything works.
```
from deephaven.types import *
```

Reply:
Confirmed.  Just restarted everything and it works for all of the types.